process/transaction/metaProcess.go

```diff
@@ -20,6 +21,16 @@ type metaTxProcessor struct {
 	txTypeHandler process.TxTypeHandler
 }
 
+// GetSmartContractResults returns the smart contract results
+func (txProc *metaTxProcessor) GetSmartContractResults() map[string]data.TransactionHandler {
+	return make(map[string]data.TransactionHandler)
+}
+
+// SetAccountsAdapter updates the accounts adapter
+func (txProc *metaTxProcessor) SetAccountsAdapter(accounts state.AccountsAdapter) {
```

Comment:
no need for this set.

Reply:
good catch. this resulted in overwriting the read-only accounts db into the real one